test/js/node/http/node-http.test.ts

```diff
@@ -219,6 +219,36 @@ describe("node:http", () => {
       });
     }
 
+    it("should be async readable", async done => {
```

Comment:
this test doesn't work 

the builtins were never re-generated, which means the changes to the JS were not reflected, but this test did not fail